src/components/YourCompany/YourCompany.vue

```diff
@@ -10,11 +10,11 @@
       <label class="define-company-title">Your Company</label>
     </div>
 
-    <div class="section-container mt-4">
+    <div id="company-name-section" class="section-container" :class="{'invalid-section': invalidNameSection}">
```

Comment:
does this create two class attributes or is vue smart?

Reply:
Vue is pretty smart :) 

It boils down too `class= "section-container invalid-section" if the condition is met in the class binding. 
I imagine it works similar to `classList.add()`

Reply:
great